packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -189,9 +201,219 @@ const renderQueryEditorHeaderLabel = (
+                placeholder="Search a query by name"
+              />
+              {!editorStore.searchText ? (
+                <div className="query-editor__query-loader__search-section__input__search__icon">
+                  <SearchIcon />
+                </div>
+              ) : (
+                <>
+                  <button
+                    className="query-editor__query-loader__search-section__input__clear-btn"
+                    tabIndex={-1}
+                    onClick={clearQuerySearching}
+                    title="Clear"
+                  >
+                    <TimesIcon />
+                  </button>
+                </>
+              )}
+            </div>
+          </div>
+          <div className="query-editor__query-loader__body">
+            {!editorStore.isLoadingQueryInProgress && (
+              <>
+                {editorStore.queries.length > 0 && (
+                  <>
+                    <table className="table">
+                      <thead style={{ position: 'sticky', top: -1 }}>
+                        <tr>
+                          {queryLoaderColumns.map((column, idx) => (
+                            <th
+                              key={column}
+                              style={{ fontWeight: 'bold' }}
+                              className="table__cell--left"
+                            >
+                              {column}
+                            </th>
+                          ))}
+                        </tr>
+                      </thead>
+                      <tbody>
+                        {(editorStore.isMineOnly
+                          ? editorStore.queries.filter(
+                              (q) => q.isCurrentUserQuery,
+                            )
+                          : editorStore.queries
+                        ).map((q) => (
+                          <tr
+                            key={q.id}
+                            className={clsx(
+                              'query-editor__query-loader__body__table__row',
+                              {
+                                'query-editor__query-loader__body__table__row--selected':
+                                  selectedQueryID === q.id,
+                              },
+                            )}
+                            onClick={(event) => setSelectedQueryID(q.id)}
+                          >
+                            {buildQueryTableRow(q).map((value, columnIdx) => (
+                              // eslint-disable-next-line react/no-array-index-key
+                              <td key={columnIdx} className="table__cell--left">
+                                {value}
+                              </td>
+                            ))}
+                          </tr>
+                        ))}
+                      </tbody>
+                    </table>
+                    <div className="query-editor__query-loader__body__warning">
```

Comment:
Didn't we discuss about this? I think for now we could remove this message. The way I think about this is, if we are to show this message at all, we need to do a little more work. Let's say, you are trying to show 10 queries, you need to query for 11, if 11 comes back, you can show 10 and show this warning. But that's some work to do, @MauricioUyaguari what do you think about this?

Reply:
I deleted this section for now.